web-ng/src/app/components/card-view-project/card-view-project.component.html

```diff
@@ -0,0 +1,41 @@
+<!--
+  Copyright 2021 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<div fxLayout="column">
+  <gnd-header-layout> </gnd-header-layout>
```

Comment:
Add comment here instead of empty space.

Reply:
updated